docs/src/getting-started/train_yaml_config.rst

```diff
@@ -0,0 +1,381 @@
+.. _train_yaml_config:
+
+Training YAML Reference
+************************************
+Overview
+===================
+``metatrain`` uses a YAML file to specify the parameters for model training,
+accessed via ``mtt train options.yaml``. In this section, we provide a complete reference
+for the parameters provided by the training YAML input. For a minimal example of a YAML input file, suitable to start a first training, we refer the viewer to the sample YAML file in the :ref:`Quickstart <label_quickstart>` section.
+
+The YAML input file can be divided into five sections:
+
+- Computational Parameters
+- :ref:`Architecture <available-architectures>`
+- loss
+- data
+- wandb integration
```

Comment:
I thought we link within this document so just giving the titles. Maybe the syntax is different

```suggestion
- :ref:`Computational Parameters`
- :ref:`Architecture`
- :ref:`loss`
- :ref:`data`
- :ref:`wandb integration`
```